plugin/manager_test.go

```diff
@@ -147,6 +147,20 @@ func (s *ManagerSuite) TestInitializePlugin_noOpIfEmpty() {
 	assert.Nil(s.T(), s.manager.loadPlugins(""))
 }
 
+func (s *ManagerSuite) TestInitializePlugin_noOpIfDotFile() {
+	tmpDir := test.NewTmpDir("gotify_testinitializeplugin_dotfile")
+	defer tmpDir.Clean()
+	os.Mkdir(tmpDir.Path(".test"), 0o755)
```

Comment:
The test implies it's a file, but it's a directory here? Should this be a create file?